cloudinary/auth_token.py

```diff
@@ -41,6 +42,7 @@ def _digest(message, key):
 
 
 def _escape_to_lower(url):
-    escaped_url = quote_plus(url)
-    escaped_url = re.sub(r'%..', lambda x: x.group(0).lower(), escaped_url)
+    from cloudinary.utils import smart_escape
```

Comment:
What is the rationale of importing smart_escape here, and not globally? it is always called either way

Reply:
auth_token is used in utils, so there is a circular dependency. And different python versions handle it differently. Local import is the safest bet.